frontend/src/api/k8s/pvcs.ts

```
@@ -115,7 +125,23 @@ export const updatePvc = (
   if (!data.description && pvcResource.metadata.annotations?.['openshift.io/description']) {
     pvcResource.metadata.annotations['openshift.io/description'] = undefined;
   }
+  const modelNameTrimmed = data.modelName?.trim() ?? '';
+  const modelPathTrimmed = data.modelPath?.trim() ?? '';
 
+  if (pvcResource.metadata.annotations) {
+    // Model Name
+    if (modelNameTrimmed) {
+      pvcResource.metadata.annotations['dashboard.opendatahub.io/model-name'] = modelNameTrimmed;
+    } else {
+      delete pvcResource.metadata.annotations['dashboard.opendatahub.io/model-name'];
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)